### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by WingChroniam** [PokeAlert/PokeAlert#100]
Can we add a scanning location queue? Before a certain update I was able to scan another nearby area and when it finishes up it starts scanning in my default location again. However I noticed you made the change to stop background scanning first so I can manually scan a specific area. Though I understand this is a good step to avoid too many scans, it's only a slight inconvenience to make sure to go back and re-enable the background scanning.

**Comment by PokeAlert** [comment]
Yep, having the background scan run at the same time as the manual brought quite some problems I had to stop temporarily the service and the restart. it is asynchron and not working for evey device the same. it was to complex to keep doing it that way

**Comment by WingChroniam** [comment]
Do you think adding a queue would be possible?

**Comment by MrMushimoto** [comment]
Would it be possible to set particular accounts to scan a particular area and have the app process those accounts/locations automatically?
For example, I want to track 3-4 small locations that are a few miles apart, and I don't want to seem like I am teleporting. If I could assign 7 accounts to track place X (or even 1 account that performs the sweep for that area), 1 account to track place Y, and a final account to track place Z, I could accomplish my goals without seeming like I'm teleporting and the app can notify me when something appears in one of the spots.

**Comment by PokeAlert** [comment]
Multi "lock" scan is on my list todo as well
